docs/references/sdk-metrics.mdx

```diff
@@ -163,6 +164,15 @@ from the queue.
 - Available in: Core, Go, Java
 - Tags: `namespace`, `task_queue`
 
+### activity_succeed_endtoend_latency
+
+Total latency of successfully finished Activity Executions from the time they are scheduled to the time they are completed.
+This metric is not recorded for async Activity completion.
+
+- Type: Histogram
+- Available in: Go, Java
```

Comment:
I am not sure if Go reports this metric (and Core). Java does. 
I can figure it out if you want.

Reply:
@antmendoza Yes please do! This is from my search yesterday:

![image](https://github.com/temporalio/documentation/assets/156466156/cef1edc1-0ebb-45b9-bb05-97ed0f1f7e78)